net/url/test.cc

```diff
@@ -105,11 +109,73 @@ TEST(URLTest, RedirectPreservesProtocolHostAndPortTest) {
             URL("blah://new_host:6000/foo", URL("meh://localhost:5000")).ComposeURL());
 }
 
+TEST(URLTest, ExtractsURLParameters) {
+  {
+    URL u("www.google.com");
+    EXPECT_EQ("", u.fragment);
+    EXPECT_EQ("", u["key"]);
+    EXPECT_EQ("default_value", u("key", "default_value"));
```

Comment:
Re: `u("key", "default_value")`
Nice but frustrating to call a URL as a function.